cartography/rules/data/frameworks/mitre_attack/requirements.py

```diff
@@ -0,0 +1,36 @@
+"""
```

Comment:
I think that most frameworks will require custom queries in their associated Facts. To avoid a combinatorial explosion of very similar-looking Facts in the community_facts dir, it would be good to include Framework specific Facts within the files that include the requirement. This will make requirements files (oof Python collision) quite long. As such, it would be good to put each Requirement in it's own file. The __init__ for the given framework can be untouched.